client/signup/steps/website-content/section-generator.tsx

```diff
@@ -1,28 +1,70 @@
 import { CONTACT_PAGE } from 'calypso/signup/difm/constants';
+import {
+	ContactPageDetails,
+	FeedbackSection,
+	LogoUploadSection,
+} from 'calypso/signup/steps/website-content/section-types';
 import { LOGO_SECTION_ID } from 'calypso/state/signup/steps/website-content/reducer';
 import { WebsiteContent } from 'calypso/state/signup/steps/website-content/schema';
-import { ContactPageDetails } from './contact-page-details';
-import { LogoUploadSection } from './logo-upload-section';
-import { CONTENT_SUFFIX, PageDetails } from './page-details';
+import { CONTENT_SUFFIX, DefaultPageDetails } from './section-types/default-page-details';
 import type {
 	AccordionSectionProps,
 	SectionGeneratorReturnType,
 } from 'calypso/signup/accordion-form/types';
 import type { TranslateResult } from 'i18n-calypso';
 
-const getPageDetailsComponentFromPageId = ( pageId: string ) => {
+interface SectionProcessedResult {
+	sectionsDetails: Array< {
+		title: TranslateResult;
+		component: React.ReactElement;
+		showSkip: boolean;
+		validate?: () => { result: boolean; errors: Record< string, TranslateResult | null > };
+	} >;
+	elapsedSections: number;
+}
+
+const generateLogoSection = (
```

Comment:
This DIFF is shown quite weirdly but I have only added a feedback section. This logo section already existed.